src/globus_sdk/services/groups/client.py

```diff
@@ -1,8 +1,161 @@
+from enum import Enum
+from typing import Optional, Sequence
+
 from globus_sdk import client
 
 from .errors import GroupsAPIError
 
 
+class Role(Enum):
+
+    member = "member"
+    manager = "manager"
+    admin = "admin"
+
+
+class GroupMemberVisibility(Enum):
+
+    members = "members"
+    managers = "managers"
+
+
+class GroupVisibility(Enum):
+
+    authenticated = "authenticated"
+    private = "private"
+
+
+class RequiredSignupFields(Enum):
+    institution = "institution"
+    current_project_name = "current_project_name"
+    address = "address"
+    city = "city"
+    state = "state"
+    country = "country"
+    address1 = "address1"
+    address2 = "address2"
+    zip = "zip"
+    phone = "phone"
+    department = "department"
+    field_of_science = "field_of_science"
+
+
+class BatchMembershipActions:
+    """
+    An object used to represent a batch action on memberships of a group.
+    `Perform actions on group members
+    <https://groups.api.globus.org/redoc#operation/group_membership_post_actions_v2_groups__group_id__post>`_.
+
+    .. automethodlist:: globus_sdk.GroupsClient
+    """
+
+    def __init__(self):
+        self._payload: dict = {}
+
+    def accept_invites(self, identity_ids: Sequence[str]):
```

Comment:
Because `str` is a `Sequence[str]`, I think these methods should check `isinstance(..., str)` before iterating.

It's not a declared/valid usage to pass a single string, but sooner or later someone would pass a single string, `mypy` would pass, but then the result would be a call with an "unexpected" bad payload.